### PR TITLE
Pull out used types of cw721

### PR DESCRIPTION
**Pull request by Buckram123**
This PR aims to pull out used types for nft ownership from cw721 package.
### Checklist

- [ ] CI is green.
- [ ] Changelog updated.

### CODE REVIEW

File: framework/packages/abstract-std/src/objects/ownership/cw721.rs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    pub expires: Expiration,
+}
+
+#[cw_serde]
+pub struct OwnerOfResponse {
+    /// Owner of the token
+    pub owner: String,
+    /// If set this address is approved to transfer/send the token as well
+    pub approvals: Vec<Approval>,
+}
```
Comment (Buckram123) [inline framework/packages/abstract-std/src/objects/ownership/cw721.rs:31]:
```suggestion
#[cw_serde]
pub struct OwnerOfResponse {
    /// Owner of the token
    pub owner: String,
}
```
We only care about owner right now, should we skip deserializing approvals?

Comment (Buckram123) [reply framework/packages/abstract-std/src/objects/ownership/cw721.rs:31]: BTW https://github.com/CosmWasm/cosmwasm/blob/3b15bfba6406cac2f807b9af93077c04d27e79c5/CHANGELOG.md?plain=1#L157

Comment (CyberHoward) [reply framework/packages/abstract-std/src/objects/ownership/cw721.rs:31]: ahhh nice! Yeah then we can keep it out but leave it in as a commented value. 
```rust
pub struct OwnerOfResponse {
	    /// Owner of the token
	    pub owner: String,
	    /// If set this address is approved to transfer/send the token as well
	    // Left out as an optimization.
	    // pub approvals: Vec<Approval>,
	}
```

**Review by CyberHoward — APPROVED** [review]

File: framework/packages/abstract-std/src/objects/ownership/cw721.rs (hunk 1 above)
Comment (Buckram123) [reply framework/packages/abstract-std/src/objects/ownership/cw721.rs:31]: Sure!

**Pull request merged**